java/org/contikios/mrm/AngleInterval.java

```diff
@@ -295,11 +295,9 @@ public String toString() {
     String retString = "";
     for (var subInterval : subIntervals) {
       if (!retString.equals(""))
-        retString = retString.concat(" && ");
+        retString = retString + " && ";
 
-      retString = retString.concat("(");
-      retString = retString.concat(Math.toDegrees(subInterval.getLow()) + " -> " + Math.toDegrees(subInterval.getHigh()));
-      retString = retString.concat(")");
+      retString += "(" + Math.toDegrees(subInterval.getLow()) + " -> " + Math.toDegrees(subInterval.getHigh()) + ")";
```

Comment:
Since you are changing the code anyway, I think a better approach would be to use a StringBuilder?